pkg/server/server.go

```diff
@@ -3,17 +3,18 @@ package server
 import (
 	"encoding/json"
 	"fmt"
+	"github.com/coreos/dex/api"
```

Comment:
Deps sorting again.

Some of these may be able to be removed when you move some code to your package.

Reply:
fixed https://github.com/openshift/console/pull/3826/commits/028957ef237401b6560830576595e84263996061